core/src/main/scala/org/apache/spark/resource/ResourceUtils.scala

```diff
@@ -362,8 +362,13 @@ private[spark] object ResourceUtils extends Logging {
 
   def logResourceInfo(componentName: String, resources: Map[String, ResourceInformation])
     : Unit = {
+    val resourceInfo = if (resources.isEmpty) {
+      s"No extra custom resources configured for $componentName."
```

Comment:
nit: What does the `extra` mean here? Can we just mention `No custom resources` here?